### DISCUSSION TITLE
Bind TX16s 4in1 Module with Heng long tanks

### DISCUSSION THREAD

**Discussion by Gruenbaer** [pascallanger/DIY-Multiprotocol-TX-Module#716] [Q&A]
Hi, it appears that Heng Long uses a proprietary RF Protocol. Is that true?
Is there a way to bind them with a TX16s?

Thanks!

Emil

**Comment by pascallanger** [comment]
All of the manufacturers have a proprietary RF Protocol.
To add a protocol, I need access to the TX and RX/Model.

**Reply by Streifenhornwf** [reply]
Hi, i have the same Problem. Transmitter Jumper t16 with 4in1 Module and heng long tk6

**Reply by pascallanger** [reply]
To add a protocol, I need access to the TX and RX/Model.

**Comment by Streifenhornwf** [comment]
Am 15.11.2022 11:37 schrieb pascallanger ***@***.***>:
To add a protocol, I need access to the TX and RX/Model.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
And how do you get access? Is this possible online, for example via teamviewer?

**Reply by pascallanger** [reply]
I need to have the hardware sent to me n France.
